calico/felix/fiptables.py

```diff
@@ -608,8 +636,15 @@ def _stub_drop_rules(chain):
 def extract_unreffed_chains(raw_save_output):
     """
     Parses the output from iptables-save to extract the set of
-    unreferenced chains, which should be safe to delete.
+    unreferenced chains which are safe to delete.
     """
+    # PLW: This method is only called by load_unreferenced_chains.
```

Comment:
Idea was to make this dead-easy to UT by making it a pure function.  That's the only reason it's split out.